src/main/resources/config/hibernate.cfg.xml

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE hibernate-configuration PUBLIC
+    "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+    "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<hibernate-configuration>
+    <session-factory>
+        <!-- JDBC Database connection settings -->
+        <property name="connection.driver_class">com.mysql.jdbc.Driver</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/lesson27_jdbc?useSSL=false</property>
```

Comment:
Do these properties corresponds your DB settings (DB name, username and password)?